section10/script.js

```diff
@@ -0,0 +1,196 @@
+"use strict";
+
+const bookings = [];
+
+function createBooking(flightNum, passengersNum = 1, price = 199) {
```

Comment:
Magic numbers, fix it here and in multiple places
![AmbitiousPracticalJabiru-small](https://user-images.githubusercontent.com/4405297/120087175-799cca00-c0ee-11eb-83dd-2490d9982070.gif)